assesspy/tests/test_metrics.py

```diff
@@ -14,15 +14,45 @@ def metric_val(self, metric, ccao_data, quintos_data):
             return getattr(ap, metric)(*quintos_data)
         return getattr(ap, metric)(*ccao_data)
 
-    def test_metric_value_is_correct(self, metric, metric_val):
+    def test_metric_value_is_correct_ccao(self, metric, metric_val):
         expected = {
             "cod": 17.81456901196891,
             "prd": 1.0484192615223522,
             "prb": 0.0024757,
             "mki": 0.794,
             "ki": -0.06,
         }
-        assert pt.approx(metric_val, rel=0.02) == expected[metric]
+        assert pt.approx(metric_val, rel=0.01) == expected[metric]
+
+    def test_metric_value_is_correct_iaao(self, metric, iaao_data):
+        if metric in ["mki", "ki"]:
+            return None
+        else:
+            table_name, estimate, sale_price = iaao_data
+            result = getattr(ap, metric)(estimate, sale_price)
+            expected = {
+                "1_1": {
+                    "cod": 29.8,
+                    "prd": 0.98,
+                    "prb": 0.232,
+                },
+                "1_4": {
+                    "cod": 14.5,
+                    "prd": 0.98,
+                    "prb": 0.135,
+                },
+                "d_1": {
+                    "cod": 7.5,
+                    "prd": 1.027,
+                    "prb": -0.120,
+                },
+                "d_2": {
+                    "cod": 7.8,
+                    "prd": 1.056,
+                    "prb": -0.011,
+                },
+            }
```

Comment:
**[Thought, non-blocking]** If we _did_ decide to factor out an `iaao_table_name` fixture, it might make sense to move these definitions into that fixture so that the table names and their expected values are defined in one place. But again, I think this is probably fine, especially if we don't anticipate needing to reference these values in other tests.